languages/javascript/exercises/concept/array-destructuring/array-destructuring.spec.js

```diff
@@ -0,0 +1,86 @@
+import {
+  discardTopCard,
+  getFirstCard,
+  getSecondCard,
+  insertFaceCards,
+  swapTopTwoCards,
+} from './array-destructuring'
+
+describe('array-destructuring', () => {
+  describe('getFirstCard', () => {
+    const getFirstCardTestCases = [
+      [[8, 3, 9, 5], 8],
+      [[3], 3],
+    ]
+
+    getFirstCardTestCases.forEach(([deck, expected]) => {
+      test(`getFirstCard([${deck}])`, () => {
+        expect(getFirstCard(deck)).toBe(expected)
+      })
+    })
+  })
+
+  describe.skip('getSecondCard', () => {
+    const getSecondCardTestCases = [
+      [[2, 5, 1, 6], 5],
+      [[10, 4], 4],
```

Comment:
Do the simple one first. Maybe also add an array with _no_ items, and array with _1_ item.

Reply:
Done.